besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -584,6 +586,20 @@ public RunnerBuilder rpcMaxLogsRange(final Long rpcMaxLogsRange) {
     return this;
   }
 
+  /**
+   * Add enode DNS configuration
+   *
+   * @param enodeDnsConfiguration the DNS configuration for enodes
+   * @return the runner builder
+   */
+  public RunnerBuilder enodeDnsConfiguration(
```

Comment:
better to pass in the actual value (not optional) and do Optional.of() within this method
similar to rpcMaxLogsRange() above

Reply:
Thanks @macfarla that makes sense. I've made that change under the latest commit, along with an update to the change log.